sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIOTest.java

```diff
@@ -329,9 +329,8 @@ public boolean apply(@Nullable Row input) {
         }));
   }
 
-  private static void runReadTest(BigtableIO.Read read, List<Row> expected) {
-    TestPipeline p = TestPipeline.create();
-    PCollection<Row> rows = p.apply(read);
+  private void runReadTest(BigtableIO.Read read, List<Row> expected) {
+    PCollection<Row> rows = p.apply(read.getTableId() + "_" + read.getKeyRange(), read);
```

Comment:
No sure I follow.

Multiple calls to `p.apply(read)` will create a multi-rooted tree which runs pretty much the same as a single-rooted one, right? This is basically why this test still passes after this change.

Am I missing something?